app/features/information/InformationTab.js

```diff
@@ -61,60 +61,63 @@ export function InformationTab() {
 
   const hackathon = state.hackathon;
 
-  // create button blocks for "Join Slack", "View Event site", etc
-  const buttonBlock = hackathon.blocks.find(
-    (e) => e.slug && e.slug === "info-button-links"
-  );
+  // // create button blocks for "Join Slack", "View Event site", etc
```

Comment:
i think this is one of the specific things where we need to have some diversity in the format of our blocks